src/views/mfa-verify/PushForm.js

```diff
@@ -29,6 +31,18 @@ define(['okta'], function (Okta) {
 
     initialize: function () {
       this.enabled = true;
+      this.listenTo(this.options.appState, 'change:isSuccessResponse',
```

Comment:
@rchild-okta We don't want to save any changes before MFA `SUCCESS`, thus we are not modifying cookie on user clicks. Seems `rememberDevice` [approach] (https://github.com/okta/okta-signin-widget/commit/7ad708db07845d89a8ad2cb84fac1d424b3a57e0#diff-324bfcc07b8afb141c2419766be33d92R182) is updating cookie on push button click, that means if a user want to disable auto push, simply uncheck the box wouldn't work, b/c he has to `DENY` the notification first. I'm not sure this is the best user experience. So now wondering what might be the best place to handle successful MFA, any suggestion?